### ISSUE TITLE
Evaluation with Android's TalkBack

### ISSUE THREAD

**Issue by gtrogdon** [braxhawk/Blackjack#22]
Good work. I tested your game with TalkBack on Android, and it performed well overall.
The scaling of the page worked out for mobile, and functionality with TalkBack was more than functional even for a less experienced screen reader user like myself.
There were a few caveats though. First, TalkBack was unable to read the cards out properly. It would say, for example, "Two of--" and then stop, as it was unable to read the card glyphs. If there is a way to create "alt text" for these glyphs that could be read out by a screen reader, that would be excellent.
Additionally, if it were me, I would work in the Web TTS API as a backup, as it seems there was trouble with Windows users. It's likely that this game would work with more comprehensive screen readers like NVDA (since it sounds like your other open issues are about Narrator), but it's always good to allow users to opt into a back up TTS system, especially since web TTS is relatively simple to implement. And if it isn't useful to your user, just have it disabled. This suggestion is not so much my saying "do this thing now," but more of a "maybe think about this if you ever write another accessible application." 

Again, good job! Accessible games are very important.

**Comment by natasha-v** [comment]
Thank you for your feedback! We have edited the alt text to hopefully be compatible with your device now. Web TTS is a great suggestion that we are definitely looking into!
